include/asterisk/chanvars.h

```diff
@@ -33,27 +33,158 @@ struct ast_var_t {
 
 AST_LIST_HEAD_NOLOCK(varshead, ast_var_t);
 
+
```

Comment:
The additions of these variable functions doesn't appear related to group vars at all... maybe pull that out as a separate improvement change?